packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -73,19 +74,19 @@ export const IcaAccountHolderI = M.interface('IcaAccountHolder', {
     holder: M.any(),
   }),
   getPublicTopics: M.call().returns(TopicsRecordShape),
-  delegate: M.callWhen(ChainAddressShape, AmountArgShape).returns(
-    M.undefined(),
-  ),
-  redelegate: M.callWhen(
+  delegate: M.call(ChainAddressShape, AmountArgShape).returns(VowShape),
+  redelegate: M.call(
     ChainAddressShape,
     ChainAddressShape,
     AmountArgShape,
-  ).returns(M.undefined()),
-  withdrawReward: M.callWhen(ChainAddressShape).returns(
-    M.arrayOf(ChainAmountShape),
+  ).returns(VowShape),
+  withdrawReward: M.call(ChainAddressShape).returns(
+    NetworkShape.Vow$(M.arrayOf(ChainAmountShape)),
+  ),
+  withdrawRewards: M.call().returns(
+    NetworkShape.Vow$(M.arrayOf(ChainAmountShape)),
```

Comment:
I created this ticket to clean up interface guards for watchers: https://github.com/Agoric/agoric-sdk/issues/9611

TLDR; agree we should be importing something less verbose from `@agoric/vow`. We also might just go with `VowShape` instead of `Vow$` since the unwrapped shape will likely never be reached. I can't find a link atm but this topic came up in a recent PR review cc @turadg